roles/keycloak/tasks/rhsso_cli.yml

```diff
@@ -0,0 +1,20 @@
+---
+- name: Ensure required params for CLI have been provided
+  ansible.builtin.assert:
+    that:
+      - query is defined
+    fail_msg: "Missing required parameters to execute CLI."
+    quiet: true
+
+- name: Verify server management interface is functional
+  ansible.builtin.uri:
+    url: "{{ keycloak_management_url }}"
+  register: result
+  until: result.status == 200
+  retries: 5
+  delay: 5
+
+- name: "Execute CLI query: {{ query }}"
```

Comment:
@guidograzioli as this collection depends on wilfly, will not reuse the jboss_cli.yml from it?

Reply:
I see now that you are using the --out-json. I think we actually need to enhance jboss_cli to use it too and returns directly json.

Reply:
That was the initial idea of importing the eap role, but no matter what i tried, the jboss_cli distributed with keycloak/rhsso does not support to spit out json, so I am in the process of parsing DMR instead